modules/clickhouse/src/test/java/org/testcontainers/junit/clickhouse/SimpleClickhouseTest.java

```diff
@@ -1,26 +1,44 @@
 package org.testcontainers.junit.clickhouse;
 
+import io.vertx.core.Future;
+import io.vertx.core.Vertx;
+import io.vertx.mysqlclient.MySQLConnectOptions;
+import io.vertx.mysqlclient.MySQLConnection;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
 import org.junit.Test;
+import org.testcontainers.containers.ClickHouseInit;
 import org.testcontainers.containers.ClickHouseContainer;
-import org.testcontainers.db.AbstractContainerDatabaseTest;
 
-import java.sql.ResultSet;
-import java.sql.SQLException;
+import java.util.concurrent.CountDownLatch;
 
-import static org.rnorth.visibleassertions.VisibleAssertions.assertEquals;
-import static org.testcontainers.ClickhouseTestImages.CLICKHOUSE_IMAGE;
+import static org.junit.Assert.assertFalse;
 
-public class SimpleClickhouseTest extends AbstractContainerDatabaseTest {
+public class SimpleClickhouseTest {
 
     @Test
-    public void testSimple() throws SQLException {
-        try (ClickHouseContainer clickhouse = new ClickHouseContainer(CLICKHOUSE_IMAGE)) {
-            clickhouse.start();
+    public void testRawMysql() throws Throwable {
+        ClickHouseContainer clickHouseContainer = new ClickHouseContainer();
+        clickHouseContainer.start();
```

Comment:
Why did you remove the try-with-resource? We use this to close the container after the test.

Reply:
Fixed